app/models/fullcalendar_engine/event_series.rb

```diff
@@ -1,7 +1,7 @@
 module FullcalendarEngine
   class EventSeries < ActiveRecord::Base
     
-    attr_accessor :title, :description, :commit_button
+    attr_accessor :title, :description, :commit_button, :color
```

Comment:
You simply have to take care of the events create through the event_series part.
Just add another entry for color [here](https://github.com/vinsol/fullcalendar-rails-engine/blob/master/app/models/fullcalendar_engine/event_series.rb#L20).